runtests.py

```diff
@@ -39,8 +39,8 @@ def run_test_suite(args):
         USE_TZ=True,
         DATABASES={
             "default": {
-                "ENGINE": "django.db.backends.postgresql_psycopg2",
-                "NAME": "djstripe",
+                'ENGINE': 'django.db.backends.sqlite3',
```

Comment:
sqlite is a great replacement for testing if you use it in memory (with :memory:), but in our case, we suggest using postgres in production, and therefore must test against it locally. This change should be reverted.